google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/AppEngineServerConfigurationType.java

```diff
@@ -114,6 +114,6 @@ public AppServerIntegration getIntegration() {
   @NotNull
   @Override
   public String getId() {
-    return "GoogleAppEngineDevServer";
+    return "gcp-app-engine-local-run";
```

Comment:
Ideally this would be a static final member to be re-used by `AppEngineRunConfigurationConverter`, but since this class is in the Ultimate module and the converter is not (it probably should be - but I'm leaving this for a separate PR) I can't do this.